pkg/app/daemon/daemon.go

```diff
@@ -90,6 +91,15 @@ func getDaemonKind() DaemonKind {
 	return DaemonKind(intValue)
 }
 
+func getCommentChar() byte {
+	cmd := secureexec.Command("git", "config", "--get", "--null", "core.commentChar")
+	if output, err := cmd.Output(); err == nil && len(output) == 2 {
```

Comment:
Should it be logged in case `err != nil`?

Reply:
No. The command fails when the config is unset, but that's a normal situation and we don't want to log an error about it. (We have [too many cases](https://github.com/jesseduffield/lazygit/issues/2745#issuecomment-1608880925) already where we log errors that occur in normal usage, and that's confusing.)